foxglove/cmd/import.go

```diff
@@ -26,38 +26,66 @@ func executeImport(baseURL, clientID, deviceID, deviceName, filename, token, use
 	if err != nil {
 		return err
 	}
+
+	return nil
+}
+
+func importFromEdge(baseURL, clientID, token, userAgent, edgeRecordingID string) error {
+	client := console.NewRemoteFoxgloveClient(
+		baseURL, clientID,
+		token,
+		userAgent,
+	)
+	_, err := client.ImportFromEdge(console.ImportFromEdgeRequest{}, edgeRecordingID)
+	if err != nil {
+		return err
+	}
+
 	return nil
 }
 
 func newImportCommand(params *baseParams, commandName string) (*cobra.Command, error) {
 	var deviceID string
 	var deviceName string
+	var edgeRecordingID string
 	importCmd := &cobra.Command{
 		Use:   fmt.Sprintf("%s [FILE]", commandName),
 		Short: "Import a data file to Foxglove Data Platform",
-		Args:  cobra.ExactArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
-			filename := args[0] // guaranteed length 1 due to Args setting above
-			if deviceName == "" && deviceID == "" {
-				dief("Must specify either --device-id or --device-name")
+			if deviceName == "" && deviceID == "" && edgeRecordingID == "" {
+				dief("Must specify either --device-id, --device-name, or --edge-recording-id")
+			}
+			if deviceName != "" || deviceID != "" {
+				if len(args) == 0 {
+					dief("Filename not specified")
+				}
+				filename := args[0]
+				err := executeImport(
+					params.baseURL,
+					*params.clientID,
+					deviceID,
+					deviceName,
+					filename,
+					viper.GetString("bearer_token"),
+					params.userAgent,
+				)
+				if err != nil {
+					fatalf("Failed to import %s: %s\n", filename, err)
```

Comment:
I guess I hadn't realized we have both fatalf and dief. Those do the same thing, it looks like the only difference is dief inserts the newline for you. We should probably just use that here as well, and on line 79-80.